packages/components/example-button/package-lock.json

```diff
@@ -25,7 +25,7 @@
 		},
 		"core-js": {
 			"version": "1.2.7",
-			"resolved": "https://registry.npmjs.org/core-js/-/core-js-1.2.7.tgz",
+			"resolved": "http://registry.npmjs.org/core-js/-/core-js-1.2.7.tgz",
```

Comment:
Could you please remove this change from this PR?

Reply:
Sure thing - but just to check first - I'm on NPM v6.4.1 and Node v8.11, so everyone who runs `npm install` and has the same version will end up editing the file like this...

1. Why does this happen? Is it that it was produced from a machine that was on different versions, or have we manually done a find-and-replace of http -> https (I remember talks of a bug...?)
2. Is there anything we can do other than "remember not to commit example-button/package-lock.json" on all future PRs?

Reply:
I believe it is a known issue https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818/15 which links to https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285 

The reason I suggested not including the change to `package-lock.json` for `example-button` here, is that it's unrelated to the rest of this PR.

Reply:
So ideally we should move the package-lock change to a separate PR where this can be properly documented. 👍 

Done in https://github.com/BBC-News/psammead/pull/151/commits/3f6303752f84d96b0d22da35dd82f8b6eaf69d7e